lib/GLPI/Agent/SNMP/MibSupport/Aruba.pm

```diff
@@ -83,6 +85,50 @@ sub getModel {
     return "AP $model";
 }
 
+sub run {
+    my ($self) = @_;
+
+    my $device = $self->device
+        or return;
+
+    # Get list of device ports (e.g. radioX_ssid_idY)
+    my $ports = $device->{PORTS}->{PORT};
+
+    # Equivalent to "show ap bss-table" Aruba IAP CLI output command:
+
+    # Get list of SSID
+    my $aiWlanESSIDValues = $self->walk(aiWlanESSID) || {};
+    # Get list of Radios (e.g. radioX_ssid_idY etc.)
+    my $aiWlanMACAddressValues = $self->walk(aiWlanMACAddress) || {};
+    # The list of Radios is co-related to the list of SSIDs
+    # $aiWlanMACAddressValues->{0} = radio0_ssid_id0
+    # $aiWlanESSIDValues->{0} = <SSID>
+
+    foreach my $index (keys(%$aiWlanMACAddressValues)) {
+        # Get WLAN BSSID (e.g. XX:XX:XX:XX:XX:XX)
+        my $wlanMacAddress = getCanonicalMacAddress($aiWlanMACAddressValues->{$index});
+
+        foreach my $port (keys(%$ports)) {
+            my $ifMacAddress = $device->{PORTS}->{PORT}->{$port}->{MAC};
+            next unless defined($ifMacAddress) && $ifMacAddress eq $wlanMacAddress;
+
+            my $ifDescr = $device->{PORTS}->{PORT}->{$port}->{IFDESCR};
+
+            # Defines the port alias with the name of the radio (e.g. radioX_ssid_idY)
+            $device->{PORTS}->{PORT}->{$port}->{IFALIAS} = $ifDescr;
```

Comment:
You should validate `$ifDescr` is defined and use it as alias only if not empty:
```suggestion
            my $ifDescr = $device->{PORTS}->{PORT}->{$port}->{IFDESCR} // "";

            # Defines the port alias with the name of the radio (e.g. radioX_ssid_idY)
            $device->{PORTS}->{PORT}->{$port}->{IFALIAS} = $ifDescr
                unless empty($ifDescr);
```
This way, as you always guaranty it is defined, you won't have perl error in later checks in the case it isn't defined.